docs/api/adbc.md

```diff
@@ -92,7 +98,8 @@ AdbcStatement adbc_statement;
 ArrowArrayStream arrow_stream;
 ```
 
-We can then initialize our database variable. Before initializing the database, we need to set two database options. The first one is the `driver`, which takes a path to the DuckDB library. The second option is the `entrypoint`, which is an exported function from the DuckDB-ADBC driver that initializes all the ADBC functions. Once we have configured these two options, we can optionally set the `path` option, providing a path on disk to store our DuckDB database. If not set, a memory database is created. After configuring all the necessary options, we can proceed to initialize our database.
+We can then initialize our database variable. Before initializing the database, we need to set the `driver` and `entrypoint` variables as mentioned above. Then we set the `path` option and initialize the database.
```

Comment:
1) instead of "variables", I'd use the term "options" to be in line with the function name `AdbcDatabaseSetOption`
2) please emphasize in the text that the `libduckdb` file should be the dynamic library of DuckDB, which is `.dylib` on MacOS, `.so` on Linux, and `.dll` on Windows.

Reply:
updated